addon/components/task-button.hbs

```diff
@@ -0,0 +1,21 @@
+<WorkItemButton
```

Comment:
That should only render if there is such a work item

Reply:
Added if conditional